src/Exception/NotFoundHttpException.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Akeneo\Pim\Exception;
+
+/**
+ * Exception thrown when a resource has not been found (404).
+ *
+ * @author    Laurent Petard <laurent.petard@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class NotFoundHttpException extends HttpException
```

Comment:
extends `ClientErrorException`. Same thing with 400 and 422 exception please, I did some mistakes...